#157552378 Create landing page

Efosaok opened the pull request:
#### What does this P.R do?
Create an intuitive landing page for the nazgul Reaction commerce app

#### Background context of task to be completed?
<img width="1440" alt="screen shot 2018-05-21 at 2 29 04 pm" src="https://user-images.githubusercontent.com/24938186/40310242-07852ecc-5d04-11e8-8381-4f7e582663ca.png">

Use Necessary css framework to create a landing page for the reaction commerce app.

#### Relevant pivotal tracker stories?
#157552378

#### snapshots?
<img width="1440" alt="screen shot 2018-05-21 at 2 29 38 pm" src="https://user-images.githubusercontent.com/24938186/40310037-72dbb8ea-5d03-11e8-9657-449a3c11fd0b.png">
<img width="1440" alt="screen shot 2018-05-21 at 2 29 25 pm" src="https://user-images.githubusercontent.com/24938186/40310151-c1790f70-5d03-11e8-80c8-ce00b9d0489c.png">
<img width="1440" alt="screen shot 2018-05-21 at 2 28 58 pm" src="https://user-images.githubusercontent.com/24938186/40310427-a919e21e-5d04-11e8-81ed-adf0ec764f8a.png">

Efosaok committed:
create custom landing page for reaction commerce app

Efosaok committed:
fix merge conflicts

Efosaok committed:
fix merge conflicts

Efosaok committed:
fix merge conflicts

Efosaok committed:
Merge branch 'feature/156958153/create-new-reaction-theme' into feature/157552378/create-landing-page

Efosaok committed:
fix alignment issues and color scheme compliance

iidrees commented:
@Efosaok You have conflict, resolve it as soon as you can so the PR can be ready for merge by the time you've regularised things on your end.

Efosaok committed:
fetch tag from datatbase

FaithAdekunle committed:
feat(shop ratings and reviews):
- Created component for displaying shop ratings and reviews
- Created components for rating and reviewing shop
- Added rate us/shop reviews tab to navbar
- Wrote tests for shop ratings and reviews
[Finishes #156958156]

FaithAdekunle committed:
resolved merge conflict

FaithAdekunle committed:
changed test parameter

FaithAdekunle committed:
resolve hound issues

FaithAdekunle committed:
fixed product ratings div styling issues

FaithAdekunle committed:
installed missing modules react-onclickoutside react-nouislider

Efosaok committed:
implement feedback on landing page

Efosaok committed:
fix merge conflicts

Efosaok committed:
align materrials with color schemes

Efosaok committed:
change footer and styling of landing page

Efosaok committed:
refactor review div section

Efosaok committed:
refactor review div section

Efosaok committed:
refactor review div section
